gen/xyz/openbmc_project/Network/StaticRoute/meson.build

```diff
@@ -13,3 +13,17 @@ generated_sources += custom_target(
     ],
 )
 
+subdir('Create')
+generated_others += custom_target(
+    'xyz/openbmc_project/Network/StaticRoute/Create__markdown'.underscorify(),
+    input: [ '../../../../../yaml/xyz/openbmc_project/Network/StaticRoute/Create.interface.yaml',  ],
+    output: [ 'Create.md' ],
+    depend_files: sdbusplusplus_depfiles,
+    command: [
+        sdbuspp_gen_meson_prog, '--command', 'markdown',
+        '--output', meson.current_build_dir(),
+        '--tool', sdbusplusplus_prog,
+        '--directory', meson.current_source_dir() / '../../../../../yaml',
+        'xyz/openbmc_project/Network/StaticRoute/Create',
+    ],
+)
```

Comment:
CI is failing because there is not an empty line at the end of this file. Your upstream gerrit link has the empty line feed.